Replicator/Replicator.cc

```diff
@@ -817,6 +820,11 @@ namespace litecore { namespace repl {
             MessageIn *response = progress.reply;
 
             if (response->isError()) {
+                if (response->getError().message == "No handler for BLIP request") {
+                    auto error = C4Error::printf(LiteCoreDomain, kC4ErrorRemoteError,
+                                                 "No handler for BLIP request 'getCollections'");
+                    return onError(error);
```

Comment:
It would be better if gotError(response) works but if not, I think there is also gotError(error) as well.

Reply:
`gotError(error)` just does an extra logging call, then calls `onError()`. The logs were very bloated when I used `gotError`, because it's already logged about 4 times, without the extra log.